src/components/dialogs/network-modifications/line-split-with-voltage-level/line-split-with-voltage-level-illustration.tsx

```diff
@@ -0,0 +1,118 @@
+    illustration: {
+        backgroundColor: 'transparent',
+        backgroundImage: 'none',
+        boxShadow: '0',
+        padding: '0px 0px',
+    },
+};
+
+/**
+ * This component displays a hide able explanation illustration
+ */
+const LineSplitWithVoltageLevelIllustration = () => {
+    const [showDiagram, setShowDiagram] = useState(true);
+    const intl = useIntl();
+    const theme = useSelector((state: ReduxState) => state[PARAM_THEME]);
+
+    useEffect(() => {
+        // dynamically modify the text inside the svg :
+        const replacedTexts = [
+            {
+                eltId: 'voltageLevelOrigBeforeTxt',
+                tradId: 'VoltageLevel1',
+            },
+            {
+                eltId: 'voltageLevelExtrBeforeTxt',
+                tradId: 'VoltageLevel2',
+            },
+            {
+                eltId: 'voltageLevelOrigAfterTxt',
+                tradId: 'VoltageLevel1',
+            },
+            {
+                eltId: 'voltageLevelToSplitAtAfterTxt',
+                tradId: 'VoltageLevelToSplitAt',
+            },
+            {
+                eltId: 'voltageLevelExtrAfterTxt',
+                tradId: 'VoltageLevel2',
+            },
+            {
+                eltId: 'lineToSplitBeforeTxt',
+                tradId: 'LineToSplit',
+            },
+            {
+                eltId: 'line1AfterTxt',
+                tradId: 'Line1',
+            },
+            {
+                eltId: 'line2AfterTxt',
+                tradId: 'Line2',
+            },
+        ];
+
+        replacedTexts.forEach((replacedText) => {
+            const elt = document.getElementById(replacedText.eltId);
+            if (elt != null) {
+                elt.textContent = intl.formatMessage({
+                    id: replacedText.tradId,
+                });
+            }
+        });
+    });
+
+    useEffect(() => {
+        /**
+         * updates opacity according to the theme by updating the 'style' string from the svg element
+         * @param propStr : String preceding and describing the property whose opacity must be changed
+         * @param svgElt : element fetched from the svg that should be updated
+         */
+        function updateOpacity(propStr: string, svgElt: HTMLElement | null) {
+            if (svgElt != null) {
+                const indexOpacity = svgElt.style.cssText.indexOf(propStr);
+                if (indexOpacity !== -1) {
+                    svgElt.style.cssText =
+                        svgElt.style.cssText.substring(
+                            0,
+                            indexOpacity + propStr.length
+                        ) +
+                        (theme === 'Light' ? '0' : '1') +
+                        svgElt.style.cssText.substring(
+                            indexOpacity + propStr.length + 1
+                        );
+                }
+            }
+        }
+        updateOpacity('fill-opacity: ', document.getElementById('background'));
```

Comment:
Initially that was my point to make it generic but you are right : it’s risky and if we have several illustration displaying in the same time it will cause many problems. Changed.

I updated the texts ids as well to follow the same nomenclature.